kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -511,15 +512,15 @@ def exec_kast(
     schedule: str,
     mode: str,
     chainid: int,
-    target: KompileTarget | None = None,
+    target: DistTarget | None = None,
     save_directory: Path | None = None,
     **kwargs: Any,
 ) -> None:
     if target is None:
-        target = KompileTarget.LLVM
+        target = DistTarget.LLVM
 
     _ignore_arg(kwargs, 'definition_dir', f'--definition: {kwargs["definition_dir"]}')
-    kevm = KEVM(target.definition_dir, use_directory=save_directory)
+    kevm = KEVM(target.check(), use_directory=save_directory)
```

Comment:
I am not sure how I feel about `_.check()` as the name, because it really just checks if the path exists. Is the intention to eventually check the integrity/up-to-date of the build as well, and roll that into this `_.check()` call?

Reply:
Good point, renamed to `get`.

> Is the intention to eventually check the integrity/up-to-date of the build as well

Yes, this should be done (ideally by the generic `kbuild` feature).